util/fibers/uring_proactor.cc

```diff
@@ -310,12 +311,37 @@ SubmitEntry UringProactor::GetSubmitEntry(CbType cb, uint16_t submit_tag) {
   return SubmitEntry{res};
 }
 
-int UringProactor::RegisterBuffers(const struct iovec* iovecs, unsigned nr_vecs) {
-  return io_uring_register_buffers(&ring_, iovecs, nr_vecs);
+int UringProactor::RegisterBuffers(size_t size) {
+  size = (size + UringBuf::kAlign - 1) / UringBuf::kAlign * UringBuf::kAlign;
+
+  // Use mmap to create the backing. For cleanup we rely just on process termination.
```

Comment:
why rely on process termination? it may create problems with unit tests, where we create/destroy lots of proactors.
at least clean it up in the d'tor